### DISCUSSION TITLE
Add a Download Command

### DISCUSSION THREAD

**Discussion by 0x7d8** [PurpurMC/Purpur#983] [__Dismissed__] [answered]
Many People have really slow internet and cant download / upload files easily.
So my Idea is:

Add a Command /download -directlink-

So the Server is used for Downloading Files directly
But so that Server Hosters wont complain, they can Add an Argument like -disabledownload to disable the feature if they dont want it.

**Comment by katekerllenevich** [comment, marked as answer]
This is more in the scope of a plugin, not in Purpur itself.

Also a security exploit waiting to happen.
